storehaus-mongodb/src/main/scala/com/twitter/storehaus/mongodb/MongoStore.scala

```diff
@@ -86,31 +89,30 @@ class MongoStore[K: MongoValue, V: MongoValue: Manifest] (
 
   override def get(key: K): Future[Option[V]] = futurePool {
     col.findOne(MongoDBObject(keyName -> key)).flatMap { valueObject => {
-        getValue(valueObject)
+        getValue(valueObject).toOption
       }
     }
   }
 
-  protected def getValue(valueObject: MongoDBObject): Option[V] = {
+  protected def getValue(valueObject: MongoDBObject): Try[V] =
```

Comment:
I think we want `Try[Option[V]]` here actually. Some things are legitimately `None`.